src/site/stages/build/README.md

```diff
@@ -0,0 +1,43 @@
+[Metalsmith Build](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/index.js#L95) Steps
+
+1. [Preserving Webpack build output](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/index.js#L59)
+2. [Create React pages](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/plugins/create-react-pages.js#L5)
+3. [Get Drupal content](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/drupal/metalsmith-drupal.js)
+4. [Add Drupal Prefix](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/plugins/add-drupal-prefix.js)
+5. [Create Outreach Assets Data](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/plugins/create-outreach-assets-data.js)
+6. [Create environment filter](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/plugins/create-environment-filter.js)
+7. [Add filenames for debugging](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/index.js#L134)
+   * Uses [`metalsmith-filenames`](https://github.com/MoOx/metalsmith-filenames)
+8. [Check collections](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/plugins/check-collections.js)
+9.  [Group collections](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/index.js#L137)
+10. [Reset left rail navigation menu levels](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/plugins/left-rail-nav-reset-levels.js)
+11. [Add the date to filenames](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/index.js#L139)
+    * Uses [`metalsmith-date-in-filename`](https://github.com/sanx/metalsmith-date-in-filename)
+12. [Add app assets](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/index.js#L140)
+13. [Add content assets](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/index.js#L141)
+14. [Plug the content into the templates](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/index.js#L162)
+    * Uses [`metalsmith-in-place`](https://github.com/metalsmith/metalsmith-in-place)
+15. [Translate the markdown to html](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/index.js#L11)
```

Comment:
Totally only stumbled on this by clicking around, but I think this is a more relevant link to the step:

https://github.com/department-of-veterans-affairs/vets-website/blob/26707cf63777c9ff34f57da0d41a0fd995fa14d5/src/site/stages/build/index.js#L165-L171